apps/pie-storybook/stories/pie-breadcrumb.stories.ts

```diff
@@ -37,6 +39,21 @@ const breadcrumbStoryMeta: BreadcrumbStoryMeta = {
             description: 'The navigation items to display in the breadcrumb. Should be an array of objects containing `label` and `href` i.e: `{ label: \'homepage\', href: \'/\' }`',
             control: 'object',
         },
+        variant: {
+            description: 'Optional variant for styling the breadcrumb component.`default` will display a regular breadcrumb component. `scrim` will enable a scrim overlay for readability.',
```

Comment:
For consistency, we use the following message for the variant prop. Each variant is usually described on the component’s overview page in pie.design so we don't have to do it here.

```suggestion
            description: 'Set the variant of the breadcrumb.',
```